README.md

```diff
@@ -137,7 +137,8 @@ W4G128 Average Accuracy of 13 tasks and Time Cost Results(Testing was conducted
 | Light   | 0.4052(2m)       | 0.5108(3m)      | **0.6453**(5m) | 0.6104(6m)      | 0.6063(6m)      | 0.7243(37m)      |
 
 
-
+<details>
+  <summary>W2G64 results</summary>
 W2G64 Average Accuracy of 13 tasks and Time Cost Results(Testing was conducted on the Nvidia A100 80G using the version of PyTorch 2.6.0 with enable_torch_compile):
```

Comment:
add this in the end.  We recommend using higher precision for the head, tail, and non-expert modules to alleviate the significant accuracy drop.